### DISCUSSION TITLE
is it possible to disable scientiffic notation on axis?

### DISCUSSION THREAD

**Discussion by ForberichN** [epezent/implot#339] [Q&A] [answered]
Currently, any value higher than 999,999 will be displayed in scientific notation ie 1e+06. Is it possible to change that behavior to display the values normally?

**Comment by epezent** [comment, marked as answer]
See `SetupAxisFormat` -- you can either provide a custom format specifier string (the default is `%g`) or a custom formatter callback. These are demonstrated in `implot_demo.cpp` under `ShowDemo_TickLabels`.

**Reply by ForberichN** [reply]
Thank you for the quick answer.
